model/implicit_constraints.py

```diff
@@ -89,16 +111,31 @@ def add_mapping_shift_to_demand(self, x, y):
                 quicksum(
                     quicksum(
                         x[e, t_marked, v] for t_marked in self.combined_time_periods
-                        if t - v + 1 <= t_marked <= t
+                        if t - v + self.time_step <= t_marked <= t
+                        if (e,t_marked,v) in x
+                        # 
+                        # and
+                        # t_marked + v - self.time_step in self.combined_time_periods
                     )
                     for v in self.shift_durations["work"]
-                )
-                == quicksum(y[c, e, t] for c in self.competencies if y.get((c, e, t)))
+                ) == quicksum(y[c, e, t] for c in self.competencies if y.get((c, e, t)))
                 for e in self.employees
                 for t in self.combined_time_periods
+                #for t in self.combined_time_periods if t < 165.75
             ),
             name="mapping_shift_to_demand"
         )
+        """
+        self.model.addConstrs(
+            (
+                x[e, 165.75, 2.25]
+                == quicksum(y[c, e, t] for c in self.competencies if y.get((c, e, t)))
+                for e in self.employees
+                for t in self.combined_time_periods if t >= 165.75
+            ),
+            name="mapping_shift_to_demand_special_case"
+        )
+        """
```

Comment:
Should this be deleted @sondrenaerland ?

Reply:
Fjern!